client/package.json

```diff
@@ -53,6 +54,7 @@
     ]
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
```

Comment:
@existentialcoder : 
Any particular reason why we have included this package? 
And also babel-eslint is now deprecated in favour of @babel/eslint-parser
(https://github.com/babel/babel-eslint#note-babel-eslint-is-now-babeleslint-parser-and-has-moved-into-the-babel-monorepo)

Reply:
I got an error when tried to run stating this module is not found

Reply:
Will check this again